app/views/traces/new.html.erb

```diff
@@ -3,7 +3,8 @@
 <% end %>
 
 <%= bootstrap_form_for @trace, :url => { :action => "create" }, :html => { :multipart => true } do |f| %>
-  <%= f.file_field :gpx_file, :placeholder => t("helpers.file.prompt") %>
+  <%= f.file_field :gpx_file, :placeholder => t("helpers.file.prompt"),
+                              :help => ".gpx, .tar.gz, .tar.bz2, .tar, .zip, .gpx.gz, .gpx.bz2" %>
```

Comment:
Are you not putting the help string into `en.yml` [help scope](https://github.com/openstreetmap/openstreetmap-website/blob/0423d18a8f5b232aa2a08b55351bee553047fb81/config/locales/en.yml#L153) on purpose, for it not to show up in translations because there's nothing to translate?

Reply:
Yes, I couldn't imagine how I could translate list of file formats. But, if you think we should put it, I can put it?